docker-entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
```

Comment:
Quoting from the referenced issue:

> Some people use tricks to read these files and export to environment variables, but it's not recommended:
>
> Docker secrets do not set environment variables directly. This was a conscious decision, because environment variables can unintentionally be leaked between containers (for instance, if you use --link).

Shall we read the file from nodejs app instead?

Reply:
adding native support, ptal: https://github.com/supabase/postgres-meta/pull/337